src/app/components/home-page/performance-slider/performance-slider.component.html

```diff
@@ -35,7 +35,7 @@ <h3 class="performance-title">{{ data.title }}</h3>
     <div class="container__button">
       <a [href]="data.ticket" class="button-ticket tickets-button" i18n="@@buyTicket">Buy ticket</a>
     </div>
-    <span class="age-restrictions">{{ data.ageLimit }}+</span>
+    <span *ngIf="data && data.ageLimit + 1" class="age-restrictions">{{ data.ageLimit }}+</span>
```

Comment:
```data.ageLimit + 1```
What does this condition mean?

Reply:
We can have `ageLimit = 0` and in this case, condition will be false. With this condition, we always have false if value NaN

Reply:
I think condition `*ngIf="data && (data.ageLimit || data.ageLimit === 0)"` should be more readable in this case